### PR TITLE
Choose files in flash request screen

### PR DESCRIPTION
**Pull request by bzapa**

### CODE REVIEW

File: app/src/main/java/programatorus/client/screens/debugger/DebuggerFragment.kt [hunk 1]
```diff
@@ -0,0 +1,49 @@
+package programatorus.client.screens.debugegr
```
Comment (bgrzesik) [inline app/src/main/java/programatorus/client/screens/debugger/DebuggerFragment.kt]: Typo

File: app/src/main/java/programatorus/client/screens/debugger/messagelist/MessageListItem.kt [hunk 2]
```diff
@@ -0,0 +1,11 @@
...
+
+data class MessageListItem(
+    val message: String,
+    val isLocal: Boolean
+) {
+    companion object {
+        fun local(message: String) = MessageListItem(message, true)
+        fun remote(message: String) = MessageListItem(message, false)
+    }
+}
```
Comment (bgrzesik) [inline app/src/main/java/programatorus/client/screens/debugger/messagelist/MessageListItem.kt]: `sealed class/interface` would be better suited here. But you can leave it.

**Review by bgrzesik — DISMISSED** [review]

File: app/src/main/java/programatorus/client/screens/debugger/DebuggerFragment.kt (hunk 1 above)
Comment (bzapa) [reply app/src/main/java/programatorus/client/screens/debugger/DebuggerFragment.kt]: fixed

File: app/src/main/java/programatorus/client/screens/debugger/messagelist/MessageListItem.kt (hunk 2 above)
Comment (bzapa) [reply app/src/main/java/programatorus/client/screens/debugger/messagelist/MessageListItem.kt]: Leaving as it is

**Review by bgrzesik — APPROVED** [review]

**Pull request merged**